File: backend/src/main/java/com/lawencon/ticketms/dto/ticket/TicketInsertResDto.java
package com.lawencon.ticketms.dto.ticket;

public class TicketInsertResDto {
	private TicketInsertDataResDto data;
	private String message;
	public TicketInsertDataResDto getData() {
		return data;
	}
	public void setData(TicketInsertDataResDto data) {
		this.data = data;
	}
	public String getMessage() {
		return message;
	}
	public void setMessage(String message) {
		this.message = message;
	}
}

File: frontend/src/app/dto/ticket/ticket.insert.res.dto.ts
import { TicketInsertDataResDto } from "./ticket.insert.data.res.dto";

export class TicketInsertResDto{
    private _data! : TicketInsertDataResDto
	private _message! : string

    public get data(): TicketInsertDataResDto {
        return this._data;
    }

    public set data(data: TicketInsertDataResDto) {
        this._data = data;
    }

    public get message(): string {
        return this._message;
    }

    public set message(message: string) {
        this._message = message;
    }

}